src/main/java/step2/ManualNumberGenerator.java

```diff
@@ -0,0 +1,24 @@
+package step2;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class ManualNumberGenerator implements LottoNumberGenerator{
```

Comment:
```suggestion
public class ManualNumberGenerator implements LottoNumberGenerator {
```
중괄호 앞은 한칸 띄는게 컨벤션 입니다. 다른 코드들도 검토해 볼까요?